tianshou/data/batch.py

```diff
@@ -29,6 +29,30 @@ def _is_batch_set(data: Any) -> bool:
     return False
 
 
+def is_scalar_value(value: Any) -> bool:
+    # check if the value is a scalar
+    # 1. python bool object, number object: isinstance(value, Number)
+    # 2. numpy scalar: isinstance(value, np.generic)
+    # 3. python object rather than dict / Batch / tensor
+    # the check of dict / Batch is omitted because this only checks a value.
+    # a dict / Batch will eventually check their values
+    is_scalar = isinstance(value, Number)
```

Comment:
Take advantage of short circuiting.

Reply:
short circuiting is okay. I write this because the line is too long, so I break it into multiple lines. Efficiency should not be a issue, though. Short circuiting still works.

Reply:
Indeed. Ok then.